### PR TITLE
Update to the about & activities pages

### PR DESCRIPTION
**Pull request by DStinDiess**

### CODE REVIEW

File: app/Resources/views/default/activities.html.twig [hunk 1]
```diff
@@ -61,12 +63,22 @@
...
-                <h1> Membership Program </h1>
+                <h1> Mentorship Program </h1>
 				<p>
-                    Coming soon.. 
+                    For those whose first year it is at UCR either as first year students or as transfer students we can pair you up with experienced ACM members so they can guide you as you explore our university. Help with classes, projects, and career pathways are all available with your individual mentor! 
                 </p>
                 <p>
-                    Time and Location: Coming soon
+                    Want a mentor? <a href="http://acmucr.org/mentee"> acmucr.org/mentee</a>
+                    <br />
```
Comment (Kilenaitor) [inline app/Resources/views/default/activities.html.twig]: nit: avoid `<br>`s

File: app/Resources/views/default/activities.html.twig [hunk 2]
```diff
@@ -61,12 +63,22 @@
                     <a href="http://citrushack.com">www.citrushack.com</a>
                 </p>
                 <hr />
-                <h1> Membership Program </h1>
+                <h1> Mentorship Program </h1>
 				<p>
```
Comment (Kilenaitor) [inline app/Resources/views/default/activities.html.twig]: nit: indentation

File: app/Resources/views/default/about.html.twig [hunk 3]
```diff
@@ -52,7 +52,7 @@
                 <hr />
                 <h1> When are your general meetings? </h1>
 				<p>
```
Comment (Kilenaitor) [inline app/Resources/views/default/about.html.twig]: nit: indentation

File: app/Resources/views/default/activities.html.twig [hunk 4]
```diff
@@ -44,6 +44,8 @@
                     beginners to the hackathon scene. 
                 </p>
                 <p>
+                    This year Cutie Hack will be on Nov 11th, 2017!
+                    <br />
```
Comment (Kilenaitor) [inline app/Resources/views/default/activities.html.twig]: nit: avoid `<br>`s.

File: app/Resources/views/default/activities.html.twig [hunk 5]
```diff
@@ -79,7 +91,9 @@
                     tract is open to all projects.
                 </p>
                 <p>
-                    Time and Location: Coming soon
+                    Interested? Information is <a href="https://acmucr.org/incubator">here</a>!
+                    <br />
```
Comment (Kilenaitor) [inline app/Resources/views/default/activities.html.twig]: nit: avoid `<br>`s

File: app/Resources/views/default/about.html.twig (hunk 3 above)
Comment (DStinDiess) [reply app/Resources/views/default/about.html.twig]: I don't see an indentation problem

**Comment by aarohmankad** [comment]
@DStinDiess Fix Kyle's changes and I can merge.

I would recommend using a `<small></small>` tag instead of a `<br />` tag.

File: app/Resources/views/default/activities.html.twig (hunk 1 above)
Comment (JohnPhamous) [reply app/Resources/views/default/activities.html.twig]: Agreed

File: app/Resources/views/default/activities.html.twig (hunk 5 above)
Comment (JohnPhamous) [reply app/Resources/views/default/activities.html.twig]: Ditto

File: app/Resources/views/default/about.html.twig (hunk 3 above)
Comment (JohnPhamous) [reply app/Resources/views/default/about.html.twig]: The `<p>` should be at the same level as `<h1>`.

File: app/Resources/views/default/activities.html.twig (hunk 4 above)
Comment (JohnPhamous) [reply app/Resources/views/default/activities.html.twig]: Add padding instead of `<br>`.

**Pull request merged**